features/counters.yml

```
@@ -0,0 +1,17 @@
+name: Counters (CSS)
+description: The `counter()` and `counters()` CSS functions, used with `counter-reset` and `counter-increment`, automatically number headings or ordered list items.
```

The "used with" phrasing here really confused me, as if it'd be `counter-reset: counter(…)`. I think listing things without more detail might be in our interest here:

```suggestion
description: The `counter-reset` and `counter-increment` CSS properties and the `counter()` and `counters()` functions automatically number headings or ordered list items.
```